### DISCUSSION TITLE
Issues with the highlight function

### DISCUSSION THREAD

**Discussion by gmg** [yandeu/five-server#15] [Help / Questions] [answered]
First of all, thank you for creating this. I had almost given up on a functioning Live Preview after transitioning from Brackets to VS Code. Now on to my issue/question:

When you select an element in VS-Code, the corresponding element in the browser view is higlighted briefly and receives a light blue border, similar to how it works in Brackets. This appears to be done using a data attribute in the HTML and a CSS rule with an attribute selector.

The problem is that the style rules used for the highlight have two side effects:
- The blue highlight overrides the background colour of the element, leaving it white
- The blue outline adds 1px to the size of the element on all sides

An ideal solution to this would be one that shows the margin and padding with different coloured highlights, similar to what you get with Inspect Element. Brackets appears to achieve this effect by creating a set of overlay <div> elements that are sized and positioned to match the highlighted element's box model.

**Comment by yandeu** [comment, marked as answer]
This has been fixed.
